quis/quis.json

```diff
@@ -97,13 +93,12 @@
                 }
             ]
         },
-        "CPk":
+        "_Rk":
         {
             "type": "gate",
-            "longName": null,
-            "aliases": ["cpk", "CRk", "crk"],
+            "longName" : null,
```

Comment:
I think it would be better to make this an empty string?

Reply:
Why doesn't it have a long name, btw?

Reply:
It could, of course have a long name, I haven't filled out this JSON entirely. The semantics were the focus. I guess the long name would be: "Controlled phase shift (pi/2^k)" as stated in the [cQASM spec](https://qutech-delft.github.io/cQASM-spec/latest/language_specification/instructions/gates.html#standard-gate-set)

I see now btw that I forgot to remove the "type" field.

Reply:
Shouldn't it be removed then?

Reply:
Is it safer to have an empty string, if there is no value, to avoid 'unknown key errors'?

Reply:
I meant; shouldn't the `type` be removed? For the `longName`, it should be either an empty string or a `null` value. Both have their merits.

Reply:
Yes, I will remove the `type` field. For `longName` I would argue to use `null` if the value does not exist, and not an empty string, as that suggests that it does, i.e. empty string suggest that the longName is a blank.